cannon/mipsevm/tests/evm_common_test.go

```diff
@@ -568,6 +568,139 @@ func TestEVMSingleStep_MthiMtlo(t *testing.T) {
 	}
 }
 
+func TestEVMSingleStep_BeqBne(t *testing.T) {
```

Comment:
(nit) The function names here have recently been updated to follow a new format:
```suggestion
func TestEVM_SingleStep_BeqBne(t *testing.T) {
```